pycbc/workflow/core.py

```diff
@@ -198,7 +198,10 @@ def __init__(self, cp, name,
             raise TypeError("Failed to find %s executable " 
                             "at %s" % (name, exe_path))
         
-        self.add_pfn(exe_path)
+        if exe_path.startswith('gsiftp://'):
+            self.add_pfn(exe_path,site='pycbc-code')
```

Comment:
Just so as I can understand this better, because I never really understood why this needs a site= entry at all and we do use this elsewhere (ie. remote frame files if querying a non-local backup frame server). It's a gsiftp link, you get it by running globus-url-copy and that's it. Why do you need to know anything about the remote site that using a generic site="nonlocal" doesn't fulfill? Hell, using site="local" works in that the file will get gsiscp-ed, but then if using the symlink options pegasus will try to symlink against a location that doesn't exist, and pegasus *knows* doesn't exist (to me that seems a bug, and I don't understand why you/Karan say it is not). I think I don't understand the "site=" property for gsiftp links at all, it should be irrelevant, no?

Using site='pycbc-code' implies only code.phy.syr.edu which seems restrictive.